openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -224,7 +243,7 @@ def test_get_account_default(self):
         self.client.login(username=self.user.username, password=TEST_PASSWORD)
         response = self.send_get(self.client)
         data = response.data
-        self.assertEqual(12, len(data))
+        self.assertEqual(13, len(data))
```

Comment:
Again, whenever you bump up the number of fields, you should also be testing that the profile image information is correct.